doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -138,6 +138,13 @@ LTE link control library
      * Remove the use of the ``CONFIG_LTE_NETWORK_USE_FALLBACK`` Kconfig option.
        Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
        In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
+     * If your application uses the :c:func:`lte_lc_conn_eval_params_get` function, you must enable the new :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option.
```

Comment:
```suggestion
     * If your application uses the :c:func:`lte_lc_conn_eval_params_get` function, you must use the new :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option.
```
I'd say use a Kconfig option instead of enable, in all instances.